### DISCUSSION TITLE
AzuraCast Not Embedding on Website and other

### DISCUSSION THREAD

**Discussion by micsolostudios** [AzuraCast/AzuraCast#4660] [Q&A]
Has anyone else had problems with azuracast not being able to embed on other websites? I copy the code , and paste it on my wordpress website, and it's empty.  I'm using servercheap.com to host it because they offered this 1 click install, but I'm wondering if that has something to do with it. Any answers?

**Comment by BusterNeece** [comment]
Do you have an example we could look at? There may be some setting preventing embedding from working correctly that we need to look at, but we can't do that without a real-world example of the issue.

**Reply by micsolostudios** [reply]
It's working now, I  had to use LetsEncrypt because my website has SSL and I think it was blocking it because of that. Although even when I used LetsEncrypt it seemed to take a day or so to update and allow me to start embedding.

**Comment by ivellios1988** [comment]
Maybe you should describe to us how and where you try to embed the code.
